.changeset/erpnext-initial.md

```diff
@@ -0,0 +1,14 @@
+---
+"@openfn/language-erpnext": minor
```

Comment:
This changeset file needs to be deleted (we don't include a changeset for initial release, and if it's left means that the first release will be 1.1.0)

I suppose what you could do  is copy the most important, most user-facing items into `changelog.md`, just so that we have a little more colour